app/src/main/java/com/onboarding/pronosticAR/utils/DateUtils.kt

```diff
@@ -0,0 +1,10 @@
+package com.onboarding.pronosticAR.utils
+
+object DateUtils {
+    fun getDate(date: String): String {
+        val month = date.substring(ConstantUtils.STRING_MONTH_START, ConstantUtils.STRING_MONTH_END)
+        val day = date.substring(ConstantUtils.STRING_DAY_START, ConstantUtils.STRING_DAY_END)
+        val hour = date.substring(ConstantUtils.STRING_HOUR_START, ConstantUtils.STRING_HOUR_END)
+        return "$day${ConstantUtils.SLASH}$month $hour${ConstantUtils.COLON}${ConstantUtils.STRING_MINUTES}"
```

Comment:
please avoid single use vars, you can use = instead of return

Reply:
done